protos/dcs/mission/v0/mission.proto

```diff
@@ -420,7 +445,7 @@ message StreamEventsResponse {
     ConnectEvent connect = 8192;
     DisconnectEvent disconnect = 8193;
     PlayerSendChatEvent player_send_chat = 8194;
-    PlayerChangeSlotEvent player_change_slot = 8195;
+    MissionCommandEvent mission_command = 8195;
```

Comment:
Removing `player_change_slot` was probably not intended?

Reply:
Probably a rebase cockup, I shall fix.